For whatever reason, hdl_graph_slam doesn't set the correct C++ version when
compiling for AArch64.  Fix this, and also set -march=native when compiling
for AArch64 as this brings the behaviour more in-line with that for x86.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 79e6312..c9fad40 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,8 +4,8 @@ project(hdl_graph_slam)
 
 # Can we use C++17 in indigo?
 if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
-  add_definitions(-std=c++11)
-  set(CMAKE_CXX_FLAGS "-std=c++11")
+  add_definitions(-std=c++14 -march=native)
+  set(CMAKE_CXX_FLAGS "-std=c++14 -march=native")
 else()
   add_definitions(-std=c++14 -msse -msse2 -msse3 -msse4 -msse4.1 -msse4.2)
   set(CMAKE_CXX_FLAGS "-std=c++14 -msse -msse2 -msse3 -msse4 -msse4.1 -msse4.2")
